RNTester/src/examples/GeolocationExample.re

```diff
@@ -0,0 +1,144 @@
+open BsReactNative;
+
+module GeolocationGetCurrentPositionExample = {
+  type coords = {
+    latitude: float,
+    longitude: float,
+  };
+  type action =
+    | UpdateCoords(coords);
+  type state = {coords};
+  let getCurrentPosition = (_event, {ReasonReact.send}) =>
+    Geolocation.getCurrentPosition(
+      position =>
+        send(
+          UpdateCoords({
+            latitude: position##coords##latitude,
+            longitude: position##coords##longitude,
+          }),
+        ),
+      error => Js.log(error),
+    );
+  let component =
+    ReasonReact.reducerComponent("GeolocationGetCurrentPositionExample");
+  let make = _children => {
+    ...component,
+    initialState: () => {
+      coords: {
+        latitude: 0.0,
+        longitude: 0.0,
+      },
+    },
+    reducer: (action, _state) =>
+      switch (action) {
+      | UpdateCoords(coords) => Update({coords: coords})
+      },
+    render: ({state, handle}) =>
+      Style.(
```

Comment:
is it common practice to "open a local" in the middle of render instead of just doing Style. inline?

Reply:
It's just a preference, I use it a lot if I have more than 2 values from the same module and don't want to open it globally in the file. For smaller snippets I do it inline